modules/vibrantmediaBidAdapter.js

```diff
@@ -213,7 +213,9 @@ export const spec = {
    * @param {*} bidData the data associated with the won bid. See example above for data format.
    */
   onBidWon: function(bidData) {
-    logInfo('Bid won: ' + JSON.stringify(bidData));
+    if (bidData && bidData.meta && bidData.meta.wp) {
+      triggerPixel(`${bidData.meta.wp}${bidData.status}`);
```

Comment:
can you make sure biddata.wp is a url?

Reply:
bidData.meta.wp is guaranteed to be a valid url, since it's an internal endpoint we use for reporting purposes (bids/won bids ratio and similar stats), generated server-side because we use several dynamic query string parameters in it.
We could add regex-based URL validation for bidData.meta.wp in the adapter though. Would that meet your expectations?

Reply:
That works :)

Reply:
@ChrisHuie updated accordingly, thanks :)